src/store/debug/debug-actions.ts

```diff
@@ -0,0 +1,7 @@
+import { createAction } from '@reduxjs/toolkit';
+
+import { ActionArrivalPayload } from '../../interfaces/action-arrival-payload.interface';
+
+export const DEBUG_ACTIONS_NAMES = ['debug/PUSH_ACTION'];
```

Comment:
we don't have any other action at this moment, just use single variable instead of array